src/app/components/App.jsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 
-
+/**
+ * Main application component
```

Comment:
This is a personal preference, but what do you guys think of leaving an empty new line between the main description and the rest of the JSDoc? Something like:

```javascript
/**
 * Main application component
 *
 * @class
 * @classdesc This is the application entry point
 * @prop {string} welcomeMessage - contains the welcome message to be shown
 */
```

@jsainthilaire @jenriquejr